src/Products/SiteErrorLog/www/main.pt

```diff
@@ -34,7 +34,7 @@
 		<div class="form-group row">
 			<label for="id" class="form-label col-sm-5 col-md-4">Ignored exception types</label>
 			<div class=" col-sm-7 col-md-8">
-				<textarea name="ignored_exceptions:lines" class="form-control" rows="3"
+				<textarea name="ignored_exceptions:ulines" class="form-control" rows="3"
```

Comment:
This change stores `unicode` data on Python 2. Is the code of `Products.SiteErrorLog` able to run with this value as `unicode`? What happens if the user has previously stored `bytes` in this field. Can it still be rendered and saved after this change?